src/main/java/org/assertj/core/error/NoElementsShouldSatisfy.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2018 the original author or authors.
+ */
+package org.assertj.core.error;
+
+public class NoElementsShouldSatisfy extends BasicErrorMessageFactory {
```

Comment:
are you sure both methods `noElementsShouldSatisfy` are used ?
if not let's keep the only one used.

Reply:
removed unused factory and c'tor

Reply:
you removed the message I wanted to keep (the one where we show the faulty element), can you revert that and get rid of this error message instead ?